.github/workflows/deploy.yml

```diff
@@ -0,0 +1,34 @@
+name: Deploy
+
+on:
+  # https://docs.github.com/en/actions/reference/events-that-trigger-workflows
+  workflow_dispatch:
+  push:
+    # Edit to the branch(es) you want to build and deploy on each push.
+    branches: ['main']
+
+env:
+  REACT_APP_PUBLIC_SURVEY_ENABLED: 'false'
+  REACT_APP_API_URL: https://bhpa-backend-boston-heat-pump-accelerator-dbab1b.apps.shift.nerc.mghpcc.org
+
+jobs:
+  deploy-server:
+    uses: ./.github/workflows/deploy.yml
+    needs: [test-backend]
+    with:
+      app: bhpa-backend
+      context: backend
+      port: 3000
+    secrets: inherit
+  deploy-website:
+    uses: ./.github/workflows/deploy.yml
+    needs: [test-ui]
+    with:
+      app: bhpa-frontend
+      context: frontend/front
+      port: 8080
+    secrets:
+      build-args: |
```

Comment:
Not 100% positive all of the "normal" secrets are going to get passed like I expect...